File: conversions/django/species.py
class Species(AbstractElementModel):

    # Biology
    appearance = models.TextField(blank=True, null=True)
    life_span = models.PositiveIntegerField(blank=True, null=True)
    average_weight = models.PositiveIntegerField(blank=True, null=True)
    nourishment = models.ManyToManyField("Species", blank=True, related_name="species_nourishment")

    # Psychology
    instincts = models.TextField(blank=True, null=True)
    aggression = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(100)])
    agency = models.TextField(blank=True, null=True)
    languages = models.ManyToManyField("Language", blank=True, related_name="species_languages")

    # World
    impact = models.TextField(blank=True, null=True)
    habitat = models.ManyToManyField("Location", blank=True, related_name="species_habitat")
    interaction = models.ManyToManyField("Phenomenon", blank=True, related_name="species_interaction")
    consumables = models.ManyToManyField("Construct", blank=True, related_name="species_consumables")

    def __str__(self):
        return self.name

File: conversions/csharp_unity_json_properties/Species.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Utils;

[System.Serializable]
public class Species : Element
{
    [JsonProperty("appearance"), TextAttribute("")]
    public string appearance;
    [JsonProperty("life_span"), Integer(0)]
    public int lifeSpan;
    [JsonProperty("average_weight"), Integer(0)]
    public int averageWeight;
    [JsonProperty("nourishment"), ReferenceAttribute(typeof(Species), true)]
    public string nourishment;
    [JsonProperty("instincts"), TextAttribute("")]
    public string instincts;
    [JsonProperty("aggression"), Integer(100)]
    public int aggression;
    [JsonProperty("agency"), TextAttribute("")]
    public string agency;
    [JsonProperty("languages"), ReferenceAttribute(typeof(Language), true)]
    public string languages;
    [JsonProperty("impact"), TextAttribute("")]
    public string impact;
    [JsonProperty("habitat"), ReferenceAttribute(typeof(Location), true)]
    public string habitat;
    [JsonProperty("interaction"), ReferenceAttribute(typeof(Phenomenon), true)]
    public string interaction;
    [JsonProperty("consumables"), ReferenceAttribute(typeof(Construct), true)]
    public string consumables;
}
